package.json

```diff
@@ -1,11 +1,13 @@
 {
-  "name": "nextjs-static-demo",
-  "version": "0.1.1",
+  "name": "calvium-nextjs-typescript-tailwind-template",
+  "version": "0.1.2",
   "private": true,
   "scripts": {
     "dev": "NODE_OPTIONS='--inspect' next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "check-eslint": "./node_modules/.bin/eslint \"src/**/**.?(ts|tsx)\" --fix --max-warnings=0",
+    "check": "npm run check-prettier --silent && npm run check-eslint --silent && npm run check-tsc --silent && npm run test --silent"
```

Comment:
The `check` script refers to other scripts `check-prettier`, `check-tsc` and `test` that don't exist here

Reply:
thanks - you are right. I'l fix that

Reply:
done